src/main/java/com/tinkoff/edu/app/LoanCalcRepository.java

```diff
@@ -8,7 +8,7 @@
 
     private static int requestId;
 
-    public static int save() {
+    public int save() {
```

Comment:
либо же наоборот, залогировать здесь, что сохранен реквест с id 
```suggestion
public int save() {
        LoanCalcLogger.log("request with id " + requestId + " saved");
```
или
```suggestion
public int save(LoanRequest request) {
        LoanCalcLogger.log("request with id " + request.getRequestId + " saved");
```